packages/docs/docs/releases.md

```
@@ -1,5 +1,155 @@
 
+## 26.6.0
+
+Release date: 2026-06-01
+
+- TODO: Add release highlights
+
+**Docker Tag: 26.6.0**
+
+<!-- release-notes:auto-generated -->
+
+#### Features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)